DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/stylecop.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do we need this file as we already have a `stylecop.json` in root directory `DICOM`?

Reply:
Same for the one here `DeID.SharedLib.UnitTests/stylecop.json`.

Reply:
Thanks a lot, deleted repetitive files.